### PR TITLE
Fix column widths in sources table and bulk action

### PR DESCRIPTION
**Pull request by AdityaKhatri**
## Changes

* Define column widths

## This PR doesn't introduce any:

- [x] temporary files, auto-generated files or secret keys
- [x] build works
- [x] eslint issues
- [ ] typescript issues
- [x] `console.log` meant for debugging
- [x] typos
- [x] unwanted comments
- [x] conflict markers

## This PR contains valid:

- [x] permission checks
- [x] translations

>NOTE: Need to talk with @frozenhelium regarding one added typescript issue

### CODE REVIEW

**Review by subinasr — COMMENTED** [review]
image
I think the action column has some styling issue?

**Comment by AdityaKhatri** [comment]
> image
> I think the action column has some styling issue?

Yes @subinasr , I've created another issue for it: https://github.com/the-deep/client/issues/1922

**Review by subinasr — APPROVED** [review]

File: src/newViews/Tagging/Sources/SourcesTable/BulkActions/styles.scss [hunk 1]
```diff
@@ -0,0 +1,23 @@
+.bulk-actions-bar {
+    display: flex;
+    position: absolute;
+    bottom: var(--dui-spacing-extra-small);
+    left: 50%;
+    align-items: center;
+    transform: translate(-50%, 0%);
+    z-index: 1;
+    border-radius: 5px;
+    background-color: #414141;
```
Comment (tnagorra) [inline src/newViews/Tagging/Sources/SourcesTable/BulkActions/styles.scss:10]: Are we hard-coding the color values?

Comment (tnagorra) [reply src/newViews/Tagging/Sources/SourcesTable/BulkActions/styles.scss:10]: Similar comment for radius and min-width

File: src/newViews/Tagging/Sources/SourcesTable/index.tsx [hunk 2]
```diff
@@ -143,34 +171,41 @@ function SourcesTable(props: Props) {
         hideSingleSourceAddModal,
     ] = useModalState(false);
 
+    const handleDelete = useCallback((leadId: number) => {
+        deleteLead(leadId);
+    }, [deleteLead]);
+
```
Comment (tnagorra) [inline src/newViews/Tagging/Sources/SourcesTable/index.tsx:177]: Why create a separate function.

**Review by tnagorra — CHANGES_REQUESTED** [review]

**Review by tnagorra — APPROVED** [review]

**Pull request merged**